Task_Position_the_Cubes/index.css

```diff
@@ -1 +1,46 @@
 /*  Start coding below for POSITION THE CUBES TASK*/
+.parent {
+  width: 300px;
+  height: 300px;
+  /* padding: 80px 50px;
+  padding-bottom: 80px;
+  padding: left 30px;;
```

Comment:
The border property should have a space between the value and the unit (border: 5px solid gray;)